run.py

```diff
@@ -1,5 +1,7 @@
 #  -*- coding: utf-8 -*-
 
+kafka_server = "172.18.0.3:9092"
```

Comment:
I guess there is no way to further parameterise this in a way that `mantidpython` can deal with it (forward arguments)?

Reply:
I haven't tried, but I think it should forward arguments without any issues. Just never implemented argument parsing when this was written. I could have a look at that now.

Although, if I move configurable parameters to a dedicated python file as described above, that might be enough?

Reply:
Looking at your example above. Having something like:
`mantidpython run.py "172.18.0.3:9092"`
or even just being able to do would be useful
`mantidpython run.py -h` 
would be useful. Having said that, there are actually a large number of things that `run.py` could parameterise (maybe too many and it shouldn't). I'll leave it up to you, my main concern is reducing the cognitive load on the next person who wants to do something with this.

Reply:
`run.py` can receive arguments like that without any trouble, but the problem is that `preprocess.py` and `postprocess.py` are not invoked from `run.py`. They're invoked by Mantid after calling `StartLiveData`.

There's no straightforward way to pass any arguments along due to this. We'd have to do something like have `run.py` write a config file and have the process scripts load it.

So I opted for just having a shared `config.py` with all configurable options, and comments to describe them.